### ISSUE TITLE
New DB table

### ISSUE THREAD

**Issue by rafelafrance** [rafelafrance/digi_leap#17]
I need to separate the label identification from the determination of label type.

- Remove label type from the labels table
- Move the label type into its own table

This is forced by the need to separate the neural net into two neural nets, one for finding the labels and one for determining the label type. This in turn was forced by a combination of a weak (small) model and bad  training data (both inaccurate and not enough classes).
